### ISSUE TITLE
Issue upgrading kernel on cubietruck

### ISSUE THREAD

**Issue by jvalleroy** [freedombox/freedom-maker#30]
```
Preparing to unpack .../linux-image-4.2.0-1-armmp-lpae_4.2.6-1_armhf.deb ...
Unpacking linux-image-4.2.0-1-armmp-lpae (4.2.6-1) over (4.2.3-2) ...
dpkg: error processing archive /var/cache/apt/archives/linux-image-4.2.0-1-armmp-lpae_4.2.6-1_armhf.deb (--unpack):
 unable to make backup link of './boot/System.map-4.2.0-1-armmp-lpae' before installing new version: Operation not permitted
dpkg-deb: error: subprocess paste was killed by signal (Broken pipe)
```
I think this error can be avoided by changing the boot partition to ext2. We will probably need flash-kernel also, to update the boot script so the new kernel can be used.

**Comment by SunilMohanAdapa** [comment]
Yes, fat filesystem seems to be the problem.  The upstream U-boot we use seems to perfectly capable of dealing with ext file systems.

**Comment by SunilMohanAdapa** [comment]
This is now fixed with #36 .
